src/Microsoft.AspNet.Mvc.Core/ModelBindingHelper.cs

```diff
@@ -293,22 +293,19 @@ public static class ModelBindingHelper
                 ModelBinder = modelBinder,
                 ValidatorProvider = validatorProvider,
                 MetadataProvider = metadataProvider,
-                HttpContext = httpContext
-            };
-
-            var modelBindingContext = new ModelBindingContext
-            {
-                Model = model,
-                ModelMetadata = modelMetadata,
-                ModelName = prefix,
-                ModelState = modelState,
+                HttpContext = httpContext,
                 ValueProvider = valueProvider,
```

Comment:
Previously `operationBindingContext.ValueProvider` would be left `null`. Doubt that was intentional but did you have to change anything else (e.g. integration tests) as a direct result of the change here?